app/upload/views.py

```diff
@@ -55,6 +55,8 @@ def upload_document(service_id):
             'mlwr_sid': sid,
```

Comment:
The API calls this method here https://github.com/cds-snc/notification-api/blob/46f9ce23d0acc8ff52fb65fe96ac72ea3f5a40bc/app/clients/document_download.py#L32-L53 and here https://github.com/cds-snc/notification-api/blob/46f9ce23d0acc8ff52fb65fe96ac72ea3f5a40bc/app/v2/notifications/post_notifications.py#L241 in the API flow